Description: Change game paths to /var/games/zangband
Author: unknown
Origin: Debian
Last-Update: 2012-11-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: zangband-2.7.5pre1/src/init2.c
===================================================================
--- zangband-2.7.5pre1.orig/src/init2.c	2005-06-05 02:54:20.000000000 +1000
+++ zangband-2.7.5pre1/src/init2.c	2007-10-10 23:11:25.000000000 +1000
@@ -214,19 +214,19 @@
 
 	/* Build a path name */
 	strcpy(tail, "apex");
-	ANGBAND_DIR_APEX = string_make(path);
+	ANGBAND_DIR_APEX = string_make("/var/games/zangband/apex");
 
 	/* Build a path name */
 	strcpy(tail, "bone");
-	ANGBAND_DIR_BONE = string_make(path);
+	ANGBAND_DIR_BONE = string_make("/var/games/zangband/bone");
 
 	/* Build a path name */
 	strcpy(tail, "data");
-	ANGBAND_DIR_DATA = string_make(path);
+	ANGBAND_DIR_DATA = string_make("/var/games/zangband/data");
 
 	/* Build a path name */
 	strcpy(tail, "save");
-	ANGBAND_DIR_SAVE = string_make(path);
+	ANGBAND_DIR_SAVE = string_make("/var/games/zangband/save");
 
 #endif /* USE_PRIVATE_PATHS */
 
